src/ContainerRegistry/ContainerRegistry/Models/ContainerRegistryDataPlaneClient.cs

```diff
@@ -33,6 +38,8 @@ public class ContainerRegistryDataPlaneClient
         private const string _refreshTokenKey = "AcrRefreshToken";
 
         private AzureContainerRegistryClient _client;
+        private Track2.ContainerRegistryClient _client2;
```

Comment:
Suggest a more readable name
```suggestion
        private Track2.ContainerRegistryClient _track2Client;
```